R/tmb.R

```diff
@@ -415,7 +392,19 @@ h_mmrm_tmb_fit <- function(tmb_object,
 #' @param control (`mmrm_tmb_control`)\cr list of control options produced
 #'   by [h_mmrm_tmb_control()].
 #'
-#' @return List of class `mmrm_tmb`, see [h_mmrm_tmb_fit()] for details.
```

Comment:
hm but then we have duplicated documentation... I think once a user is here, then it is ok to forward them to internal helper function documentation. so I would keep this as it was.